### DISCUSSION TITLE
How do we organize series files and folder structure?

### DISCUSSION THREAD

**Discussion by pixelstuff** [Kareadita/Kavita#3253] [Ideas]
### Idea Description

The wiki seem so say that books in a series should all be at a single level under the series folder like this:

```
Library Root
  ┖── Series A
      ┖── Series A - v01.cbz
      ┖── Series A - v02.cbz
      ┖── Series A - v03.cbz
```

How are we supposed to name the cover images or OPF metadata for each book in the series? The way I'm currently organizing things, because of how Calibre does it, is like this:
```
Library Root
  ┖── Series A
      ┖── Series A - Book 1
          ┖── Series A - Book 1.epub
          ┖── cover.jpg
          ┖── metadata.opf
      ┖── Series A - Book 2
          ┖── Series A - Book 2.epub
          ┖── cover.jpg
          ┖── metadata.opf
      ┖── Series A - Book 3
          ┖── Series A - Book 3.epub
          ┖── cover.jpg
          ┖── metadata.opf
```

What would be the proper way of keeping this data with the files in a series?

### Idea Category

User Experience

### Duration of Using Kavita

1 week

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by scare376** [comment]
Covers and opf files need to be internal. Kavita does not use sidecar files

**Reply by pixelstuff** [reply]
Does Calibre automatically store metadata changes to internal files, as well as apparently generating the side car files, or is there an editor that can show what is inside the file?

**Reply by locke69321** [reply]
It does not, you can use Polish in calibre or [follow this guide for exporting epubs in calibre.](https://wiki.kavitareader.com/guides/external-tools/calibre) CBZs you should look into something not calibre, like Manga Manager or Comictagger depending on what you're tagging, and keep those separate from calibre altogether. At the end of the day, the internal metadata is more important that the library structure, so long as all the files aren't in the library root.

**Reply by scare376** [reply]
^ this. the calibre file structure can work for epubs but is not recommended and will likely not work for other file types. use the polish plugin and make sure the update metadata and cover boxes are checked off in order to keep the calibre file structure.
